Makefile.3ds

```diff
@@ -110,7 +110,7 @@ export INCLUDE  := $(foreach dir,$(INCLUDES),-I$(CURDIR)/$(dir)) \
 
 export LIBPATHS :=  $(foreach dir,$(LIBDIRS),-L$(dir)/lib)
 
-ifeq ($(strip $(ICON)),)
+ifeq ($(strip $(APP_ICON)),)
```

Comment:
I'm sorry, I intended to only have the change in the Makefile not the Makefile.3ds. I was not aware it would also grab the Makefile.3ds which I committed after making the pull request. Sorry!